Source/Extensions/Blazorise.Splitter/wwwroot/blazorise.splitter.js

```diff
@@ -0,0 +1,81 @@
+import './vendors/split.js?v=1.1.3.0';
+
+document.getElementsByTagName("head")[0].insertAdjacentHTML("beforeend", "<link rel=\"stylesheet\" href=\"_content/Blazorise.Splitter/blazorise.splitter.css?v=1.1.3.0\" />");
+
+/**
+ * Configuration options for Split
+ * @typedef {Object} SplitOptions
+ * @property {number[] | undefined} sizes - Initial sizes of each element in percents or CSS values.
+ * @property {number | number[] | undefined} minSize - Minimum size of each element.
+ * @property {number | number[] | undefined} maxSize - Maximum size of each element.
+ * @property {boolean | undefined} expandToMin
+ * @property {number | undefined} gutterSize - Gutter size in pixels.
+ * @property {'center' | 'start' | 'end' | undefined} gutterAlign -
+ * @property {number | number[] | undefined} snapOffset - Snap to minimum size offset in pixels.
+ * @property {number | undefined} dragInterval -
+ * @property {'horizontal' | 'vertical' | undefined} direction - Direction to split
+ * @property {string | undefined} cursor - Cursor to display while dragging.
+ * /
+ /**
+ * Creates a new Split instance
+ * @param  {HTMLElement} elements The elements to split
+ * @param {SplitOptions} options Configuration options
+ * @return A split instance
+ */
+export function initializeSplit(elements, options) {
+    return Split(elements, parseOptions(options));
```

Comment:
We usually call init with an elementid, keep the instance in memory to get rid of it later, etc....
Does this handle multiple split instances without conflicts?

Reply:
Yes, it works pretty well.